### ISSUE TITLE
Remmina can't connect to key based SSH server running on non-standard port

### ISSUE THREAD

**Issue by e-alfred** [FreeRDP/Remmina#984]
Hello,

I am trying to connect to a server using my ssh key running on a non-standard port. Using password-based authentication works well. but trying to connect using the key I get the following error:

> [SSH] ssh_connect: libssh 0.6.1 (c) 2003-2014 Aris Adamantiadis, Andreas Schneider, and libssh contributors. Distributed under the LGPL, please refer to COPYING file for information about your rights, using threading threads_pthread
> [SSH] socket_callback_connected: Socket connection callback: 1 (0)
> [SSH] ssh_client_connection_callback: SSH server banner: SSH-2.0-OpenSSH_6.8p1-hpn14v6
> [SSH] ssh_analyze_banner: Analyzing banner: SSH-2.0-OpenSSH_6.8p1-hpn14v6
> [SSH] ssh_analyze_banner: We are talking to an OpenSSH client version: 6.8 (60800)
> [SSH] ssh_packet_userauth_failure: Access denied. Authentication that can continue: publickey,password
> [SSH] ssh_agent_get_ident_count: Answer type: 12, expected answer: 12
> [SSH] ssh_packet_userauth_failure: Access denied. Authentication that can continue: publickey,password
> [SSH] ssh_packet_userauth_failure: Access denied. Authentication that can continue: publickey,password

Connecting to a similar system running on port 22 works well.

Thank you

**Comment by antenore** [comment]
How did you have specified the ssh port?

You should set **Server** to IP:port

It works for me.

**Comment by e-alfred** [comment]
Yes, it is. Otherwise I wouldn't be able to connect to the server by simply changing to password authentification.

**Comment by stefanct** [comment]
I have the same problem. I am pretty sure it started when updating the (ssh) server from Ubuntu 15.10 (wily) to 16.04 (xenial). FWIW, logging in with (openssh's) `ssh` and key files works just fine.

In my case it is libssh 0.6.3, OpenSSH 7.2p2 (on the server and client) and remmina 1.1.2.

**Comment by antenore** [comment]
I'm working on it, #1196  are you available for testing?
I'll commit soon the ed25519 patch

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @antenore @e-alfred @stefanct
